Homework-6/script.js

```diff
@@ -0,0 +1,262 @@
+// Task 1
+
+// const camelize = str => {
+//   let arr = str.split("-");
+//   arr = arr.map(item => {
+//     return item.charAt(0).toUpperCase() + item.slice(1);
+//   });
+//   return console.log(arr.join(""));
```

Comment:
You can chain all operations like this:
```
const camelize = str => {
  return str
    .split("-")
    .map(word => word[0].toUpperCase() + word.slice(1))
    .join("");
};
```

Don't return `console.log()`. In this case your function return `undefined`, instead of camelize string

Reply:
.map(word => word[0].toUpperCase() + word.slice(1))
                         ^
TypeError: Cannot read property 'toUpperCase' of undefined

Reply:
item.charAt(0).toUpperCase() + item.slice(1); => work!

Reply:
Разница только в том, что если символ с такой позицией отсутствует, тогда [] вернёт undefined, а charAt — пустую строку.
You are right! Artem navalil